pom.xml

```diff
@@ -122,15 +138,15 @@
       <plugins>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.5.1</version>
+          <version>${compiler.plugin.version}</version>
           <configuration>
-            <source>1.6</source>
-            <target>1.6</target>
+            <source>${java.source}</source>
```

Comment:
Probably just want "java.version" here for both source and target, since there's never a case where we're going to want the two to diverge.